README.md

```diff
@@ -271,14 +271,35 @@ $ python examples/single_agent.py scenarios/loop
 
 ```bash
 # For this to work, your account needs to be added to the huaweinoah org
-$ cd /path/to/SMARTS
+$ cd </path/to/SMARTS>
 export VERSION=v0.4.17
 docker build --no-cache -f ./utils/docker/Dockerfile -t smarts:$VERSION .
 docker tag smarts:$VERSION huaweinoah/smarts:$VERSION
 docker login
 docker push huaweinoah/smarts:$VERSION
 ```
 
+### Singularity
+```bash
+# Build container from definition file.
+$ cd </path/to/SMARTS>
+$ sudo singularity build ./utils/singularity/smarts.sif ./utils/singularity/smarts.def
+
+# Run container in interactive mode.
+$ cd </path/to/SMARTS>
+$ singularity shell --containall --bind ../SMARTS:/SMARTS ./utils/singularity/smarts.sif
+# Inside the container
+$ python3.7 /SMARTS/examples/single_agent.py /SMARTS/scenarios/loop/ --headless
```

Comment:
I think the scenarios would have to be built first?
```suggestion
# Inside the container
$ scl scenario build /SMARTS/scenarios/loop/
$ python3.7 /SMARTS/examples/single_agent.py /SMARTS/scenarios/loop/ --headless
```

Reply:
I have improved the singularity instructions.